io.catenax.mp_standard_response/1.0.0/ProvideProductionForecast.ttl

```diff
@@ -0,0 +1,128 @@
+   samm:properties ( :listOfForecastItems [ samm:property :iterationNo; samm:optional true ] ext-types:communicationMode ext-types:versionDataModel ) .
+
+:listOfForecastItems a samm:Property ;
+   samm:preferredName "list Of Forecast Items"@en ;
+   samm:description "List of ForecastItems matching the items to an order"@en ;
+   samm:characteristic :ForecastItemList .
+
+:iterationNo a samm:Property ;
+   samm:preferredName "iterationNo"@en ;
+   samm:description "only set in CommunicationMode == \"notification/cycle\" to be able to check the order of the notifications. "@en ;
+   samm:characteristic ext-types:IntegerValueCharacteristic ;
+   samm:exampleValue 6 .
+
+:ForecastItemList a samm-c:List ;
+   samm:preferredName "Forecast Item List"@en ;
+   samm:description "List with the forecast items"@en ;
+   samm:dataType :ForecastItem .
+
+:ForecastItem a samm:Entity ;
+   samm:preferredName "ForecastItem"@en ;
+   samm:description "a ForecastItem entry for the requested order"@en ;
+   samm:properties ( :positionId :productionForecast :precisionOfForecast :prodcutionStatus [ samm:property :reasons4Delay; samm:optional true ] :returnCode :forecastDate ) .
+
+:positionId a samm:Property ;
+   samm:preferredName "positionId"@en ;
+   samm:description "Identifier of a position of an order"@en ;
+   samm:characteristic ext-header:UuidCharacteristic ;
+   samm:exampleValue "00000000-0000-0000-C000-000000000046" .
+
+:productionForecast a samm:Property ;
+   samm:preferredName "production Forecast"@en ;
+   samm:description "Date of completion"@en ;
+   samm:characteristic ext-header:Timestamp ;
+   samm:exampleValue "2023-06-19T21:24:00+07:00"^^xsd:dateTimeStamp .
+
+:precisionOfForecast a samm:Property ;
+   samm:preferredName "precision Of Forecast"@en ;
+   samm:description "Accuracy of the prediction"@en ;
+   samm:characteristic ext-types:TimeValueCharacteristic .
+
+:prodcutionStatus a samm:Property ;
+   samm:preferredName "prodcution Status"@en ;
+   samm:description "Status of the order/position within MP"@en ;
+   samm:characteristic :ProductionStatusEnum ;
+   samm:exampleValue "itemPlanned" .
+
+:reasons4Delay a samm:Property ;
+   samm:preferredName "reasons 4 Delay"@en ;
+   samm:description "Optional field to provide information to the customer why a delivery date is not met"@en ;
+   samm:characteristic :Reasons4DelayEnum ;
+   samm:exampleValue "supply problems" .
+
+:returnCode a samm:Property ;
+   samm:preferredName "return Code"@en ;
+   samm:description "Return code that indicates whether a single item of an order matches the customers desired request"@en ;
+   samm:characteristic :ReturnCodeEnum ;
+   samm:exampleValue "lower accuracy of precision" .
+
+:forecastDate a samm:Property ;
+   samm:preferredName "forecast Date"@en ;
+   samm:description "Date/time of the forecast calculation"@en ;
+   samm:characteristic ext-header:Timestamp ;
```

Comment:
The model
https://github.com/eclipse-tractusx/sldt-semantic-models/pull/227/files

does not contain a characteristic Timestamp. it uses
samm-c:Timestamp ;